src/lib/worker.js

```diff
@@ -76,11 +77,15 @@ export default class Worker {
       });
 
       this.child.on('error', (err) => {
-        resolve(this.formatResults(1, startTime, err));
+        resolve(this.formatResults(1, startTime, cucumberOut + '\n' + err));
```

Comment:
I'm not sure we'd have any cucumberOut data here, this `on('error')` catches problems specifically with child process  performing some bad way, not necessarily the cucumber run within it. I initially tried to use this for the stderr output and found that it basically never gets called for the errors we commonly see. Still good to have, and don't think this will be an issue leaving this line as you have it, but just thought I'd mention it.